### ISSUE TITLE
Can't compile on Mac Os High Sierra

### ISSUE THREAD

**Issue by cinder92** [electron-userland/electron-packager#785]
Hello, after update to Mac OS High Sierra o can't compile. 

Error log

```
MacBook-Pro-de-Dante:rancherita-panel-electron dantecervantes$ electron-packager .
Downloading tmp-2696-0-SHASUMS256.txt-1.6.6
[============================================>] 100.0% of 4.44 kB (4.44 kB/s)
Packaging app for platform darwin x64 using electron v1.6.6
Command failed: npm prune --production
npm WARN rancherita-panel-electron@1.1.0 No repository field.
npm WARN rancherita-panel-electron@1.1.0 No license field.

npm ERR! May not delete: /private/var/folders/jh/5f_9xzv94zx4jqjr5kdc17h00000gn/T/electron-packager/darwin-x64/RancheritaPanel-darwin-x64/Electron.app/Contents/Resources/app/node_modules/.bin

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/dantecervantes/.npm/_logs/2018-01-03T16_46_20_471Z-debug.log
```

Any idea?

Thanks

**Comment by malept** [comment]
Thanks for filing an issue! In order to help you with your problem, we're going to need more information about it. In particular:

* Console output when you run electron-packager with the environment variable `DEBUG=electron-packager`. Please include the stack trace if one exists.

* What version of NPM is this?

* Try running `electron-packager . --no-package-manager`.

**Comment by cinder92** [comment]
Thanks for answering, at this time i fixed the issue updating my npm ;) thanks!

**Comment by malept** [comment]
@cinder92 Just so people who come across this issue know, which version of NPM did you have installed before you upgraded?

**Comment by cinder92** [comment]
latest one 5.6.0

**Comment by malept** [comment]
Sounds like a bug fixed in [NPM 5.4.1](https://github.com/npm/npm/releases/tag/v5.4.1).

**Comment by onmyway133** [comment]
I run `npm install -g npm` to update npm to 5.6.0, and the problem is fixed
